model/Security/Classes/VexAffectedVulnAssessmentRelationship.md

```diff
@@ -4,21 +4,37 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Connects a vulnerability and an element designating the element as a product
+affected by the vulnerability.
 
 ## Description
 
-An VexAffectedVulnerabilityCategorization is TODO
+`VexAffectedVulnAssessmentRelationship` connects a vulnerability and a number
+of elements. The relationship marks these elements as products affected by the
+vulnerability. This relationship corresponds to the VEX `affected` status.
 
-## Metadata
+## Constraints
```

Comment:
you can not add new headings or modify the structure of the markdown files without adopting https://github.com/spdx/spec-parser and adding the feature there. This will break the parser and the generation